Develop (#9)
qmasters-ltd/qradar-universal-cloud-rest-api: CONTRIBUTING.md

# Contributing

> At QMasters, we believe in the power of community-driven innovation. As such, we welcome and encourage contributions from the community to help improve the QRadar Universal Cloud Rest API.
> 
> We have provided a handful of workflows to get you started, but we understand that there may be specific use cases that we have not yet covered. If you have developed a workflow that you think would benefit others, we would love to hear from you!

Reviewer: ```suggestion
At QMasters, we believe in the power of community-driven innovation. As such, we welcome and encourage contributions from the community to help improve the QRadar Universal Cloud REST API.

We have provided a selection of workflows to get you started, but we understand that there may be specific use cases that we have not yet addressed. If you have developed a workflow that you think would benefit others, we would love to hear from you!
```

## Code of Conduct
We expect all contributors to adhere to our code of conduct. This includes being respectful of others and their opinions, refraining from discriminatory or abusive language, and working collaboratively to achieve our common goals.

> ## What to include:

Reviewer: Add a reference here to the source.

    - Any time we receive messages back from the endpoint we should try and capture non - authentication information into a comment
- Ensure that any complex processing is adequately commented for future maintainers / forkers to follow.
- Please Use camel casing for variable names (IE: newVariable)
- Please use indentation on opening and closing of XML tags to assist in readability, and use tab as the indentation character.
- Please try to name variables such that the names provide information about their usage - IE: "startTime" to identify the variable holding the timestamp the next query should begin at. 
- Ensure that any potentially sensitive user - provided parameters are set to secret.
- Please provide in your submission some obfuscated sample output of your workflow operating. This can be done by utilizing the workflow validation tool (https://www.ibm.com/docs/en/dsm?topic=protocol-command-line-testing-tool - with verbose output turned on).
    - **This information should be obfuscated before upload and have no PII / SPII within it**. We strictly want to see the proper operation of the workflow in the document. 

## To submit a workflow, please follow these steps:

> - Fork the QRadar Universal Cloud Rest API repository

Reviewer: ```suggestion
- Fork the [qradar-universal-cloud-rest-api](https://github.com/qmasters-ltd/qradar-universal-cloud-rest-api)
```

- Create a new branch for your changes
- Add your workflow file(s) to the appropriate folder(s)
- Commit your changes and create a pull request

> Once your pull request is reviewed and approved, we will post your workflow(s) here for others to consume. We may also feature particularly innovative or useful workflows on our website or in our marketing materials.

Reviewer: ```suggestion
Once your pull request is reviewed and approved, we will post your workflow(s) here for others to use. We may also feature particularly innovative or useful workflows on our website or in our marketing materials.
```
